versions.txt

```diff
@@ -12,3 +12,10 @@ qemu_lite_version=741f430a960b5b67745670e8270db91aeb083c5f
 mpfr_version=3.1.4
 gmp_version=6.1.0
 mpc_version=1.0.3
+cc_image_fedora_obs_version=16180-29.1
```

Comment:
Random thought: this isn't required for this PR but I think it would help if we started adding comments into this file (and maybe broke the lines into logical paragraphs :-) It is sourced so comments and whitespace shouldn't cause problems, but would make the file a lot easier to understand and maintain.